Makefile

```diff
@@ -11,4 +11,11 @@ buildroot/Makefile:
 buildroot/.config:
 	BR2_EXTERNAL=.. make -C buildroot fischertechnik_TXT_defconfig
 
+imagedir := buildroot/output/images
 
+release: all
+	$(eval version := $(shell cat buildroot/output/target/etc/fw-ver.txt))
+	$(eval zipfile := build/ftcommunity-txt-$(version).zip)
+	mkdir -p $(zipdir)
```

Comment:
$(zipdir) isn't defined anymore...

I'll merge anyway and fix this after the merge.